build.gradle.kts

```diff
@@ -38,17 +40,24 @@ if (System.getProperty("idea.sync.active").toBoolean()) {
   ideaDir.file(".name").asFile.writeText(ideName)
 
   val icon = ideaDir.file("icon.png").asFile
-  if (!icon.exists()) {
-    val img =
-      java.net
-        .URI(
-          "https://raw.githubusercontent.com/polaris-catalog/polaris/main/docs/img/logos/polaris-brandmark.png"
-        )
-        .toURL()
-        .openConnection()
-        .getInputStream()
-        .use { inp -> inp.readAllBytes() }
-    ideaDir.file("icon.png").asFile.outputStream().use { out -> out.write(img) }
+  if (!icon.exists() && !gradle.startParameter.isOffline) {
+    try {
+      val img =
+        URI(
+            "https://raw.githubusercontent.com/polaris-catalog/polaris/main/docs/img/logos/polaris-brandmark.png"
```

Comment:
Can we load from local directory(`docs/img/logos/`)?

Reply:
Also I don't think it's worth to add logic to just make IDE a little bit nicer. We can probably remove the logic completely if it fails IDE. Just my personal opinion though.

Reply:
I'd be fine with removing this download code too.

Reply:
yea - the logo's in the source-tree :facepalm: